sample/kotlin/src/main/kotlin/com/datadog/android/sample/compose/ComposeFragment.kt

```diff
@@ -85,7 +91,16 @@ fun SimpleView(
         modifier = Modifier.fillMaxSize()
     ) {
         Text("View $viewId")
-        Button(onClick = onNavigate, modifier = Modifier.padding(top = 32.dp)) {
+        val viewTreeObserver = (LocalContext.current as Activity).window.decorView.viewTreeObserver
+        Button(
+            onClick = trackClicks(targetName = "Open View", onClick = {
```

Comment:
Out of consistency with `onClick`, should we rename the method `trackClick` instead of `trackClicks`?

Reply:
sure, can do that